services.py

```
@@ -0,0 +1,301 @@
+            result = subprocess.run(
+                cmd,
+                cwd=service_path,
+                capture_output=True,
+                text=True,
+                check=False
+            )
+            
+            if result.returncode == 0:
+                return True
+            else:
+                console.print(f"[red]❌ Command failed[/red]")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)